packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -199,12 +199,21 @@ function prettyAjvErrors(errors: ErrorObject[] | null | undefined): string {
   return errors.map((e) => `${e.instancePath ?? "."} - ${e.message}`).join("\n");
 }
 
+type StringifiedProperty = string | StringifiedProperty[] | null | undefined;
```

Comment:
type still allow `null` and `undefined` but function below will always return a `string` or `string[]`

Reply:
Good catch! Fixed